mtools/util/logevent.py

```diff
@@ -653,9 +664,19 @@ def _extract_counters(self):
                         try:
                             # Remap counter to standard name, if applicable
                             counter = counter_equiv.get(counter, counter)
-                            vars(self)['_' + counter] = int((token.split(':')
-                                                             [-1]).replace(',',
-                                                                           ''))
+                            #extract allowDiskUse counter
+                            if(counter == 'allowDiskUse' and
+                                token.startswith('allowDiskUse')):
+                                try:
+                                    self._allowDiskUse = (
+                                        split_tokens[t + 1 + self.datetime_nextpos + 2].replace(',', ''))
+
+                                except ValueError:
+                                    pass
```

Comment:
is it worth to log an info message about this value error or we can suppress the error silently? assuming we log something or we print out something

Reply:
Good catch, also I'm not sure what could exactly cause a ValueError to happen here

Reply:
the split_tokens() may not return the desired value considering the key-value format is fairly inconsistent in the logs (eg. some token have spaces after ":" and some don't)

Reply:
But there's no split_tokens() function being called inside the block, see again, what's happening here inside the block is the following:

Calculation of `t+1+self.datetime_nextpos+2`

If t or self.datetime_nextpos are not an Int this raises a TypeError

Dict lookup: this raises an IndexError if it's past its size

replace: this raises an AttributeError if the result of the lookup is not a string

So, in essence nothing raises a ValueError

Now if the logs are inconsistent we need to check for the options and maybe then add catch-all exception, but right here I think this try/catch will catch anything, unless I'm missing something